pkg/subctl/cmd/deploybroker.go

```diff
@@ -56,7 +56,7 @@ var validComponents = []string{components.ServiceDiscovery, components.Connectiv
 func init() {
 	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet", false,
 		"enable support for Overlapping CIDRs in connecting clusters (default disabled)")
-	deployBroker.PersistentFlags().StringVar(&globalnetCIDRRange, "globalnet-cidr-range", "169.254.0.0/16",
+	deployBroker.PersistentFlags().StringVar(&globalnetCIDRRange, "globalnet-cidr-range", "242.254.0.0/16",
 		"GlobalCIDR supernet range for allocating GlobalCIDRs to each cluster")
 	deployBroker.PersistentFlags().UintVar(&defaultGlobalnetClusterSize, "globalnet-cluster-size", 8192,
```

Comment:
With the increase in the default Globalnet CIDR, we can even increase the default globalnet-cluster-size, but with new GN2.0 design we will not be consuming too many IPs anyway, so lets leave the default to 8192 itself.

Reply:
I think we will get more value from increasing no. of IPs in each cluster than no. of clusters that we can support, but agree that 8192 is good enough for now, we're even less likely to hit this limit with v2.

Reply:
Right, 8192 addresses should be sufficient enough for any cluster in the new implementation.